### ISSUE TITLE
libcrypt.so.1 missing

### ISSUE THREAD

**Issue by Monviech** [ValveSoftware/SteamOS#944]
#### Your system information

* Steam client version: 1671675017
* SteamOS version: 3.4.2 Build 20221221.2
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen.

Expected to happen:
- Run application in desktop mode that has a dependency on libcrypt. Application should open and run. (It did run in earlier versions of SteamOS). Example application is SecondLife Viewer.

What did happen instead:
- An error occured, the shares resource libcrypt.so.1 could not be found. The application couldnt launch. This happens since the latest stable steamos release update where arch linux got an update.

How to fix the issue manually:
- Set sudo password for deck user
- sudo steamos-readonly disable
- sudo -su root
- pacman keyring --init
- pacman keyring populate
- pacman -S libxcrypt-compat
- steamos-readonly enable

What I expect in a future SteamOS Version to fix this bug:
- supply libxcrypt-compat automatically

**Comment by ebassi** [comment]
This issue should be fixed in the latest *Main* channel, thanks for your report!

**Comment by catochondria** [comment]
Appears to not be fixed in the most recent update: 3.4.4 build 20221228.1.

Install attempts of libxcrypt-compat yield invalid signature errors no matter the method tried to install https://archlinux.org/packages/core/x86_64/libxcrypt-compat/

**Comment by bertogg** [comment]
> Appears to not be fixed in the most recent update: 3.4.4 build 20221228.1.

That's the _stable_ channel, it's only fixed in _main_ at the moment.

I can however install it manually using `pacman` without problems:

```
$ pacman-key --init
$ pacman-key --populate archlinux
$ pacman -Sy libxcrypt-compat
```

**Comment by lunarcloud** [comment]
This missing library has broken some non-steam games for me.

**Comment by bertogg** [comment]
Also fixed in SteamOS 3.5 beta
